codice/alliance
# [1.1.x] CAL-436 Add CMETAA TRE Support

#### What does this PR do?

Add support for the NITF CMETAA TRE.

#### Who is reviewing it? 
@ryeats 
@dcruver 

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@clockard
@jlcsmith 

#### How should this be tested?
This needs to be built with this imaging-nitf PR: [IMG-244 PR#190](https://github.com/codice/imaging-nitf/pull/190)
Ingest a NITF with a CMETAA TRE
Verify that the metacard has the "ext.nitf.cmetaa.*" attributes

#### What are the relevant tickets?

[CAL-436](https://codice.atlassian.net/browse/CAL-436)
[IMG-244](https://codice.atlassian.net/browse/IMG-244)

#### Checklist:
